lodash/file_content_extractor.py

```diff
@@ -0,0 +1,57 @@
+import pymupdf
+import docx
+
+def read_file_content(file_path, include_styles=False):
```

Comment:
The files would be passed as `BytesIo`, not as a `path`. In that case, file extension should be determined with `magic` package (see other prs in `core` and `llm_agency`) or accepted as an input parameter